server/doltgres_handler.go

```diff
@@ -120,11 +121,20 @@ func (h *DoltgresHandler) ComExecuteBound(ctx context.Context, conn *mysql.Conn,
 		return fmt.Errorf("boundQuery must be a sql.Node, but got %T", boundQuery)
 	}
 
+	start := time.Now()
+	if h.sel != nil {
+		h.sel.QueryStarted()
```

Comment:
Is this how we do it in Dolt too? Might be a bit misleading to call this query start when that really happens a few RPCs earlier. Do we need separate events for executing prepared statements?

Might add a TODO